runtests.py

```diff
@@ -563,23 +570,30 @@ def asv_compare_config(bench_path, args, h_commits):
         build += ["--disable-optimization"]
 
     is_cached = asv_substitute_config(conf_path, nconf_path,
-        numpy_build_options = ' '.join([f'\\"{v}\\"' for v in build]),
-        numpy_global_options= ' '.join([f'--global-option=\\"{v}\\"' for v in ["build"] + build])
-    )
+                                      numpy_build_options=' '.join(
```

Comment:
Break after the `(`
```
    is_cached = asv_substitute_config(
        conf_path,  nconf_path, . . .
```

Reply:
@charris Sorry I should add break statement in which line?